task_stages_integration_test.go

```diff
@@ -32,9 +32,9 @@ func TestTaskStagesRead(t *testing.T) {
 	rTest, rTestCleanup := createRun(t, client, wkspaceTest)
 	defer rTestCleanup()
 
-	t.Run("without include param", func(t *testing.T) {
+	t.Run("with include param to fetch the task stage of a run", func(t *testing.T) {
 		r, err := client.Runs.ReadWithOptions(ctx, rTest.ID, &RunReadOptions{
-			Include: "task_stages",
+			Include: []RunIncludeOps{RunTaskStages},
```

Comment:
These changes in this file are not relevant to this PR's purposes. But this adjustment was needed to avoid a build error pulled from releases branch